lib/APMTracer.js

```diff
@@ -22,10 +22,9 @@ import { APMSpanContext } from './APMSpanContext';
 import { ConsoleRecorder } from './recorder';
 import { AlwaysSampledSampler } from './sampler';
 
-const CARRIER_PREFIX = 'hawkular-apm-';
-const CARRIER_SPAN_ID = `${CARRIER_PREFIX}spanid`;
-const CARRIER_TRACE_ID = `${CARRIER_PREFIX}traceid`;
-const CARRIER_SAMPLED = `${CARRIER_PREFIX}sampled`;
+const CARRIER_PREFIX = 'HWKAPM';
+const CARRIER_SPAN_ID = `${CARRIER_PREFIX}ID`;
+const CARRIER_TRACE_ID = `${CARRIER_PREFIX}TRACEID`;
 
```

Comment:
There is also the business transaction and level state that will need to be received (extracted) and sent (injected) - we need to make sure all state is propagated.